encryption-formats/box2.js

```diff
@@ -0,0 +1,189 @@
+// SPDX-FileCopyrightText: 2022 Andre 'Staltz' Medeiros
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const BFE = require('ssb-bfe')
+const Ref = require('ssb-ref')
+const { isFeedSSBURI } = require('ssb-uri2')
+const { keySchemes } = require('private-group-spec')
+const { box, unboxKey, unboxBody } = require('envelope-js')
+const { directMessageKey, SecretKey } = require('ssb-private-group-keys')
```

Comment:
these 3 needs to be added to package.json

Reply:
Done!